provers/tests/src/provers/el.rs

```diff
@@ -16,16 +16,16 @@ impl<H: ZkVmHost> ElProofGenerator<H> {
 }
 
 impl<H: ZkVmHost> ProofGenerator<EvmEeProver> for ElProofGenerator<H> {
-    type Input = u64;
-    fn get_input(&self, block_num: &u64) -> ZkVmResult<EvmEeProofInput> {
-        let input = EvmSegment::initialize_from_saved_ee_data(*block_num, *block_num)
-            .get_input(block_num)
-            .clone();
-        Ok(vec![input])
+    type Input = (u64, u64);
+    fn get_input(&self, block_range: &(u64, u64)) -> ZkVmResult<EvmEeProofInput> {
+        let (start_block, end_block) = block_range;
+        let evm_segment = EvmSegment::initialize_from_saved_ee_data(*start_block, *end_block);
+
+        Ok(evm_segment.get_inputs().clone())
     }
 
-    fn get_proof_id(&self, block_num: &u64) -> String {
-        format!("el_{}", block_num)
+    fn get_proof_id(&self, block_range: &(u64, u64)) -> String {
+        format!("el_{}_{}", block_range.0, block_range.1)
```

Comment:
Should be `evmee_`.

Also can we not apply the same rework to use EVM blkids / payload IDs instead of heights here?

Reply:
Since, these test rely on [static witness data](https://github.com/alpenlabs/strata/tree/main/crates/test-utils/data/evm_ee). So, we are planning to get [rid of these](https://alpenlabs.atlassian.net/browse/STR-760) and have migrated to [functional test](https://github.com/alpenlabs/strata/blob/main/functional-tests/fn_prover_el_dispatch.py).